.github/workflows/example-repository-dispatch.yml

```
@@ -0,0 +1,180 @@
+      target_repo:
+        description: 'Target repository (optional)'
+        required: false
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      should_dispatch: ${{ steps.check.outputs.should_dispatch }}
+      event_type: ${{ steps.check.outputs.event_type }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)